doc/dev/index.rst

```diff
@@ -179,6 +179,100 @@ with::
 
 For more ``vstart.sh`` examples, see :doc:`/dev/quick_guide`.
 
+What is merged where and when ?
+===============================
+
+Commits are merged into branches according to criterions that change
+during the lifecycle of a Ceph release. This chapter is the inventory
+of what can be merged in which branch at a given point in time.
+
+Development releases (i.e. x.0.z)
+---------------------------------
+
+What ?
+^^^^^^
+
+ * features and bug fixes
+
+Where ?
+^^^^^^^
+
+Features are merged to the master branch. Bug fixes should be (but it is not mandatory)
+merged in the branch of the stable release (hammer for 0.94.x, infernalis for
+9.2.x, etc.).
```

Comment:
@jdurgin added http://tracker.ceph.com/projects/ceph-releases/wiki/HOWTO#HOWTO to the section about x.2.z since this is where it matters.